src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -57,6 +57,7 @@ private EppoClient(ConfigurationStore configurationStore, Timer poller, EppoClie
      * @param flagKey
      * @param subjectAttributes
      * @return
+     * @throws ConfigurationNotFoundException
```

Comment:
Is this necessary?

Reply:
No. In my opinion these description-less Javadocs add net negative value and should either be fleshed out, or axed.